src/validation_workflow/validation_docker.py

```diff
@@ -0,0 +1,95 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import time
+
+from validation_workflow.api_test_cases import TestCases
+from validation_workflow.check_docker_daemon import DockerDaemonRunning
+from validation_workflow.cleanup_docker import CleanupDocker
+from validation_workflow.get_local_docker_imageId import GetDockerImageId
+from validation_workflow.inspect_docker_image import InspectDockerImage
+from validation_workflow.spinup_docker_container import RunDocker
+from validation_workflow.validation_args import ValidationArgs
+
+
+class ValidationDocker():
```

Comment:
I can see there are already added abstract methods in file `src/validation_workflow/validation.py`, but not extending the class `Validation`,  (should be as `ValidationDocker(Validation)`)
This is done for the file `src/validation_workflow/validation_rpm.py`, can we follow the same please?

Reply:
> Hey @Divyaasm @jordarlu can we organize as sub folders (of distribution type) under main folder `validation_workflow`, the more distributions in future the more messy it looks and hard to debug the code, it would be easier to update/debug/add if sorted as sub folders. We can still keep the generic code out under `validation_workflow`, example for files like `src/validation_workflow/validation.py`, `src/validation_workflow/validation_args.py` but with files `validation_docker.py`, `spinup_docker_container.py` can be inside `src/validation_workflow/docker` folder. Thank you

Sure Prudhvi, that might look good altogether. We will work on it. Thanks